src/classes/Toggle.js

```diff
@@ -0,0 +1,301 @@
+import AbstractDomElement from '../AbstractDomElement'
+import DOMAnimations from '../utils/DOMAnimations'
+import uniqid from '../utils/uniqid'
+import throttle from '../utils/throttle'
+import { disableBodyScroll, enableBodyScroll } from 'body-scroll-lock'
```

Comment:
Codacy has a fix for the issue: [Replace `{·disableBodyScroll,·enableBodyScroll·}·from·'body-scroll-lock` with `uniqid·from·'../utils/uniqid`](https://app.codacy.com/gh/BeAPI/be-a11y/pullRequest?prid=8001873)
```suggestion
import uniqid from '../utils/uniqid'
```